README.md

```diff
@@ -10,7 +10,6 @@ To build it, [install Hugo](https://gohugo.io/getting-started/installing/) and [
 
 ```sh
 git clone https://github.com/OpenTermsArchive/docs
-cd docs
```

Comment:
This should stay: the `docs` directory is not _inside_ the `docs` repository, but if you `clone` from a directory, you will create a `docs` directory that contains the clone, and you need to `cd` inside it 🙂

Reply:
Oooh ok. Got it